core/google/cloud/_helpers.py

```diff
@@ -22,27 +22,18 @@
 
 import calendar
 import datetime
-import json
 import os
 import re
-import socket
 from threading import local as Local
 
+import google.auth
 from google.protobuf import timestamp_pb2
 try:
-    from google.appengine.api import app_identity
-except ImportError:
-    app_identity = None
-try:
     import grpc
 except ImportError:  # pragma: NO COVER
     grpc = None
 import six
 from six.moves import http_client
-from six.moves import configparser
-
-from google.cloud.environment_vars import PROJECT
```

Comment:
There is no equivalent for removing this. We used to have system tests that monkey-patch `_helpers.PROJECT` to get the "auto project" behavior from the environment. (Now we don't since the project comes from the `GOOGLE_APPLICATION_CREDENTIALS` file)